Description: don't always add node_modules to searched paths
Author: Julien Puydt

Forwarded: not-needed

--- a/lib/node-modules-paths.js
+++ b/lib/node-modules-paths.js
@@ -26,7 +26,7 @@
 module.exports = function nodeModulesPaths(start, opts, request) {
     var modules = opts && opts.moduleDirectory
         ? [].concat(opts.moduleDirectory)
-        : ['node_modules'];
+        : ['node_modules', ''];
 
     if (opts && typeof opts.paths === 'function') {
         return opts.paths(
